src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1125,12 +1125,28 @@ static bool _parseStyleAttr(void* data, const char* key, const char* value, bool
     sz = strlen(key);
     for (unsigned int i = 0; i < sizeof(styleTags) / sizeof(styleTags[0]); i++) {
         if (styleTags[i].sz - 1 == sz && !strncmp(styleTags[i].tag, key, sz)) {
+            string newValue = value;
+            bool importance = false;
+            if (auto ptr = strstr(value, "!important")) {
+                size_t size = ptr - value;
+                while (size > 0 && isspace(value[size - 1])) {
+                    size--;
+                }
+                newValue = string(value, 0, size);
+                importance = true;
+            }
             if (style) {
-                styleTags[i].tagHandler(loader, node, value);
-                node->style->flags = (node->style->flags | styleTags[i].flag);
+                if (importance || !(node->style->flagsImportance & styleTags[i].flag)) {
+                    styleTags[i].tagHandler(loader, node, newValue.c_str());
+                    node->style->flags = (node->style->flags | styleTags[i].flag);
+                }
             } else if (!(node->style->flags & styleTags[i].flag)) {
-                styleTags[i].tagHandler(loader, node, value);
+                styleTags[i].tagHandler(loader, node, newValue.c_str());
             }
+            if (importance) {
+                node->style->flagsImportance = (node->style->flags | styleTags[i].flag);
+            }
+
```

Comment:
if (importance) free(value);

Reply:
@hermet please check a commit with changes. I'm not sure whether it is what you wanted. I can't free a const char*. I could cast, but not sure if it;s good idea.

Before merging I'll squash the commits.

Reply:
Even that "const" we decided for ourselves. The purpose is not doing copy the text unnecesarily...  (1145)
You know what you are doing, so you can cast it & free it.